packages/@okta/vuepress-site/docs/guides/email-magic-links-overview/main/index.md

```diff
@@ -0,0 +1,244 @@
+   ```
+
+8. Click **Save** and close the dialog.
+
+Four email templates can be customized in this way. These are listed in the table below.
+
+* All support redirects via EVE and direct redirects.
+* All are compatible with applications based on either the Embedded Sign-In Widget or an Embedded SDK.
+* To add `otp` and `state` variables to the magic link, find the template you are editing in the table, and:
+  * Replace the VTL variable in the **magic link variable name** column with your custom endpoint URL.
+  * Set your `otp` query string variable to the VTL variable in the **OTP variable name** column for the template.
+  * Set your `state` query string variable to `${request.relayState}` for all templates.
+
+| Template Name  | Use Case | Magic link variable name | OTP variable name |
+|----------------|----------|--------------------------|-------------------|
+| Email challenge | Sign in with email - challenge | `${emailAuthenticationLink}`    | `${verificationToken}` |
+| Forgot Password | Self-service password recovery | `${resetPasswordUrl}`           | `${oneTimePassword}`   |
+| Registration - Activation | Self-service registration | `${registrationActivationLink}` | `${oneTimePassword}`   |
+| Email factor verification | Sign in with email - enrollment | `${verificationLink}` | `${verificationToken}` |
+
+When a user clicks the magic link based on a customized email template, their browser is redirected straight to your application's endpoint:
+
+<div class="full">
+
+![Flow diagram showing the interaction between application and Identity Engine when using custom email templates](/img/advanced-use-cases/email-magic-links-overview-custom-template-flow.png)
+
+</div>
+
+### Comparing the options
+
+|   | EVE + Default email templates | EVE + Custom email templates | Custom Email Templates |
+|---|---|----|----|
+| Set-up + Maintenance   | Requires one change in the admin console | Requires editing every email template that uses magic links to point to your new endpoint. | Requires editing every email template that uses magic links to point to your new endpoint. |
+| Speed of redirection   | Magic link sends a request to Identity Engine for the callback URL. The user is then redirected to the callback URL. | Magic link sends a request to Identity Engine for the callback URL. The user is then redirected to the callback URL. | Magic link sends browser directly to the callback URL |
+| Branding | Default Okta-branded email templates | You can add your brand to a custom email template  | You can add your brand to a custom email template |
+| Multi-language support | Default email templates support multiple languages | You must create and maintain a version of each custom template in each required language. | You must create and maintain a version of each custom template in each required language. |
+
+## See also
+
+* [Okta Email (magic link/OTP) integration guide](https://developer.okta.com/docs/guides/authenticators-okta-email/-/main/)
```

Comment:
Two changes:

1. Remove the domain names and start with the Url with "docs"
2. I understand why the tilda is used but clicking on the links bounces you out of the left nav bar which is less than optimal. Possibility default to a language?

Reply:
I've swapped out all the guide links into separate stack snippets so they all have their won correct links.